packages/vats/src/localchain.js

```diff
@@ -34,6 +34,31 @@ const { Vow$ } = NetworkShape;
  *   '[Symbol.iterator]()' method that returns an iterator.
  */
 
+/**
```

Comment:
why extract this type? it's exports a new typedef `QueryFn`. Maybe it's because you needed to export `QueryManyFn` and wanted them to match? Let's see if we can make it work without exporting that either. These functions should comply with other interfaces, not define them

Reply:
I originally intended to export all 3 since they're useful / somewhat involved. Revised just to export `QueryManyFn`. 

> Let's see if we can make it work without exporting [`QueryManyFn`] either

I don't have any ideas here, outside of reimplementing the type again. But curious to understand what you have in mind.